json_schemas/framework-agreement-details.json

```diff
@@ -3,6 +3,10 @@
   "additionalProperties": false,
   "minProperties": 1,
   "properties": {
+    "countersignerDetails": {
```

Comment:
I think since it's a key inside `framework_agreement_details` we can add additional ones beside it if we need to store more data in the future. `countersignedName` makes a little bit more sense if it's a string, I'd expect a dictionary object in `{x}Details`.